src/file-upload.js

```diff
@@ -30,3 +36,36 @@ export const attachBlobUrl = (win, file) => ({
   name: file.name,
   url: win.URL.createObjectURL(file),
 });
+
+export const FilesDragHint = ({isDisplayed}) => html`
+  <div
+    class=${classMap({
+      [g('flex-center')]: true,
+      [n('drag-hint')]: true,
+      [n('displayed')]: isDisplayed,
+    })}
+  >
+    <span class=${n('drag-hint-arrow')}>⇐</span> Drop to add files
+  </div>
+`;
+
+/**
+ * Gets files from an Event.dataTransfer object (typically from a drop.)
```

Comment:
Maybe a comment about when items vs files?

Reply:
`items` contains DOM nodes and `files`. It's not necessary for our use-case, so I'll remove this and just use `.files` directly.